### ISSUE TITLE
Setting repulsor.setForce(...) to anything other than 1 decays entity push

### ISSUE THREAD

**Issue by 19PHOBOSS98** [LubieKakao1212/OpenCombatUtility#4]
Here I've setup a spell turret (Ars Nouveau) to fire projectiles over a pulsing OpenCU repulsor. I have hitboxes enabled for better observation. Keep in mind the videos are spedup because of cmdcam+minema2 mod recording setup:

Force set to 1:
```
p=peripheral.find("opencu:repulsor")
p.recalibrateByIdx(1)
p.setRadius(5)
p.setForce(1)
p.setVector(0,1,0)

while(true) do
p.pulse(0,0,0)
sleep(0.1)
end
```

https://github.com/LubieKakao1212/OpenCombatUtility/assets/37253663/810470bf-d6e1-447b-b69d-7afba0db136a

Force set to 0.9:
```
p=peripheral.find("opencu:repulsor")
p.recalibrateByIdx(1)
p.setRadius(5)
p.setForce(0.9)
p.setVector(0,1,0)

while(true) do
p.pulse(0,0,0)
sleep(0.1)
end
```

https://github.com/LubieKakao1212/OpenCombatUtility/assets/37253663/487bd26a-27f0-4d80-a729-99d7a1a41c76

**Comment by LubieKakao1212** [comment]
fixed in 1.3.2.3

**Comment by 19PHOBOSS98** [comment]
Thank you!
